lib/rdoc/generator/template/rails/_head.rhtml

```diff
@@ -17,23 +17,6 @@
 <script src="<%= rel_prefix %>/js/searcher.js" type="text/javascript" charset="utf-8" data-turbolinks-track="reload"></script>
 <script src="<%= rel_prefix %>/panel/tree.js" type="text/javascript" charset="utf-8" data-turbolinks-track="reload"></script>
 <script src="<%= rel_prefix %>/js/searchdoc.js" type="text/javascript" charset="utf-8" data-turbolinks-track="reload"></script>
-<script type="text/javascript" charset="utf-8">
-document.addEventListener("turbolinks:load", function() {
-  // Only initialize panel if not yet initialized
-  if(!$('#panel .tree ul li').length) {
-    $('#links').hide();
-    var panel = new Searchdoc.Panel($('#panel'), search_data, tree, '<%= rel_prefix %>/');
-    $('#search').focus();
```

Comment:
This LOC was causing the refocus when I moved this over to be in `main.js`. I removed the focus @p8 in the latest version.